query-planner-js/src/buildPlan.ts

```diff
@@ -834,6 +851,11 @@ class FetchGroup {
         path: this.mergeAt!,
         node: fetchNode,
       };
+    } catch (e) {
+      console.log(`Got ${e} with ${operation}`);
```

Comment:
**question** Were these `log`s part of debugging or does it make sense to keep them in? I'm wondering if these should be changed to `.warn` or `.info` to make it more clear how bad the error is.